aten/src/ATen/native/native_functions.yaml

```diff
@@ -6112,12 +6112,13 @@
 - func: topk.values(Tensor self, int k, int dim=-1, bool largest=True, bool sorted=True, *, Tensor(a!) values, Tensor(b!) indices) -> (Tensor(a!) values, Tensor(b!) indices)
   dispatch:
     CPU: topk_out_cpu
-    CUDA: legacy::cuda::_th_topk_out
+    CUDA: topk_out_cuda
 
 - func: topk(Tensor self, int k, int dim=-1, bool largest=True, bool sorted=True) -> (Tensor values, Tensor indices)
   variants: method, function
   dispatch:
-    CPU, CUDA: topk
+    CPU: topk
+    CUDA: topk_cuda
```

Comment:
Doesn't have to be this PR, but consider making this structured https://github.com/pytorch/rfcs/blob/rfc-0005/RFC-0005-structured-kernel-definitions.md

(You actually can't make it structured yet because multi-output is not yet supported, but it will be soon)